### DISCUSSION TITLE
[Request] Better "hiding" using Bed sprite depth technology

### DISCUSSION THREAD

**Discussion by ubertpendragon** [FakeFishGames/Barotrauma#5181] [Ideas and suggestions]
Since interacting with beds puts your character at the bed's sprite depth, it's possible to hide your model behind other sprites.

_Makeshift dumpster, inspired by Hitman. Character is lying on the bed, behind the panel in the foreground_
![Capture](https://user-images.githubusercontent.com/73229309/110139731-a3866f00-7dd3-11eb-9bb5-b4df0c7885b1.JPG)

Unfortunately, the name is still visible from outside, and you can't rotate the bed to "hide" vertically, say a cabinet. It's also not possible to rename the item

Having the ability to hide names, as well as a vertical version of the item would give a new dimension to traitor gameplay by hiding in plain sight

**Comment by Ek-Videogames** [comment]
The issue of being hit by stray bullets and other attacks is also a concern.

**Comment by mLuby** [comment]
I suggest this be solved by removing the `???` (hide it or make it an empty string or space ` `) when a character is masked and not wearing an id. That way hiding in crawlspaces can also work, rather than seeing a suspicious `???` floating above the floor.
